test/unit_tests.vim

```diff
@@ -523,7 +523,7 @@ enddef
 # Test for :LspGotoDefinition, :LspGotoDeclaration and :LspGotoImpl
 def Test_LspGotoSymbol()
   silent! edit Xtest.cpp
-  sleep 200m
+  sleep 600m
```

Comment:
Is this change needed?

Reply:
yes, it happened (failed) sometimes, so extend it a bit.